data/dialogs/dialogs.py

```diff
@@ -183,6 +183,379 @@ def mod(self):
+            self.set_text(752, "Many Rage specials boost physical damage, like Stray Cat, Gold Bear, and Trooper. Try them with Sniper or Man Eater. Even Fixed Dice damage benefits from these kinds of boosts!<end>")
+            self.set_text(753, "Heard about the Serpent Trench?<line><choice> Yes<line><choice> No<end>")
+            self.set_text(754, "Jump into the Trench from Crescent Mountain.<end>")
+            self.set_text(755, "Jump into the Trench from Crescent Mountain.<end>")
+            self.set_text(756, "In the Serpent Trench, go right twice to get both chests.<end>")
+            self.set_text(757, "The Injured Lad quest can be started in the house right behind me.<end>")
+            self.set_text(758, "By default, Rage users will learn monster skills after defeating them in battle.<page>This eliminates the need to use Leap on the Veldt.<end>")
+            self.set_text(759, "Dried Meat is guaranteed to be in at least 1 shop. It can randomly show up in chests too.<page>Don't forget the airship has a shop too!<end>")
+            self.set_text(761, "The entrance to the Serpent Trench is south of here, in Crescent Mountain.<end>")
+            self.set_text(805, "If you wish, you can fight the enemy soldier NPCs in the Doma Castle siege.<page>Your party will be fully healed after you defeat the boss there.<end>")
+            self.set_text(806, "Hit the chest, don't kick it!<end>")
+            self.set_text(807, "Items sold will be at most 1/2 their shop value, depending on the seed.<end>")
+            self.set_text(809, "You can exit from this town from both the east and west.<end>")
+            self.set_text(810, "South Figaro-bound ferry.<line><choice> (No thanks)<line><choice> (Hop aboard)<end>")
+            self.set_text(812, "Nikeah-bound ferry.<line><choice> (No thanks.)<line><choice> (Hop aboard)<end>")
+            self.set_text(813, "After the Serpent Trench check, you'll end up on the docks of this town.<end>")
+            self.set_text(814, "Some towns have all 4 shop types: Item, Weapon, Armor, and Relic. This is one of them!<end>")
+            self.set_text(815, "DANCER: Yoo hoo! You handsome thing. How 'bout joining me?<line>Tee hee!<end>")
+            self.set_text(825, "Save time shopping by knowing exactly what you want to buy ahead of time.<end>")
+            self.set_text(826, "With No Priceless Items on, consider selling items for high GP values early on, like Elixirs or Genji Helmets.<end>")
+            self.set_text(917, "The reward you don't pick during the Lone Wolf check can be found in the World of Ruin Moogle Cave.<end>")
+            self.set_text(918, "With good timing you can run past the last line of soldiers during the Kefka at Narshe battle!<end>")
+            self.set_text(919, "You can't warp out of the Narshe mines in World of Balance, except for the Moogle Cave.<end>")
```

Comment:
Please remove 919, to eliminate the conflict with https://github.com/ff6wc/WorldsCollide/pull/35